core/templates/dev/head/css/oppia.css

```diff
@@ -2988,3 +2988,333 @@ md-card.preview-conversation-skin-supplemental-card {
   color: red;
   margin-bottom: 10px;
 }
+
+/*
+  about / learners / creators pages
+*/
+.clearfix {
+  display: table;
+  content: "";
+  clear: both;
+}
+
+.about-header {
+  background-color: rgb(54, 222, 207);
+  background-image: url("../images/general/icons-bg.png");
+  background-blend-mode: soft-light;
+  color: #fff;
+  text-align: center;
+  padding: 10%;
+}
+
+.about-card {
+  background-color: #fff;
+  border-radius: 5px;
+}
+
+.about-content {
+  width: 60%;
+  padding: 0;
+}
+
+.about-extra-container {
+  width: 100%;
+}
+
+.about-extra-content {
+  width: 60%;
+  margin: 0 auto;
+  position: relative;
+}
+
+.about-otter-container {
+  position: absolute;
+  display: inline-block;
+  bottom: -2.3em;
+}
+
+.otter-speech-card {
+  width: 50%;
+  float: right;
+}
+
+.otter-speech-bubble {
+  position: relative;
```

Comment:
Please alphabetize attributes within a css rule (so, background, then border-left, then border-radius, etc.). This enables a particular property's value to be found quickly, and prevents devs from specifying the same attribute multiple times in the same rule by accident.